auth/user_mgt.go

```diff
@@ -107,83 +97,236 @@ type UserIterator struct {
 
 // UserToCreate is the parameter struct for the CreateUser function.
 type UserToCreate struct {
-	params map[string]interface{}
+	_req   *identitytoolkit.IdentitytoolkitRelyingpartySignupNewUserRequest
```

Comment:
I'm not familiar with prefixing names with underscore in go. Isn't that a python convention?

Reply:
I wanted to somehow indicate in the code that this field should not be accessed directly. It should always be accessed via `request()` or `validatedRequest()` functions. This also works well with IDE assist, since typing `u.req` auto completes to `u.request()`. WDYT?